src/SDKs/CognitiveServices/dataPlane/Search/Search/Microsoft.Azure.CognitiveServices.Search.csproj

```diff
@@ -11,4 +11,7 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
@stefanvalianu you really don't need to add it (Nit)

Reply:
Also releasing without bumping up the version will fail publishing. Will need a version bump.
A minor number bump might be ok as you don't have lot of changes. As you are still in preview breaking changes does not need a major version bump.

Reply:
We haven't published yet

Reply:
@stefanvalianu ignore my comment then.